src/Validation.ContentScan.Core/Validation.ContentScan.Core.csproj

```
@@ -0,0 +1,74 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <Import Project="..\..\sign.thirdparty.props" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{912A81F7-F0D9-43EF-A589-9A341EB2D6B6}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>Validation.ContentScan.Core</RootNamespace>
+    <AssemblyName>Validation.ContentScan.Core</AssemblyName>
```

Let's align with the Validation.ScanAndSign.Core project conventions:

```suggestion
    <RootNamespace>NuGet.Jobs.Validation.ContentScan</RootNamespace>
    <AssemblyName>NuGet.Jobs.Validation.ContentScan.Core</AssemblyName>
```

See:

https://github.com/NuGet/NuGet.Jobs/blob/376ac06e6e07d4ee8d1e28f6b2346e3891487496/src/Validation.ScanAndSign.Core/Validation.ScanAndSign.Core.csproj#L11-L12